DataGrid/TreeList: add Accessibility topic (#5024)
DevExpress/devextreme-documentation: concepts/05 UI Components/TreeList/05 Accessibility/15 Keyboard Navigation.md

> An end user can use the following keys to interact with the TreeList component: 

Reviewer: Can you explain why you change it from "A user" to "An end user"?

```suggestion
A user can use the following keys to interact with the TreeList component: 
```

Author: I felt like I had to accentuate here that exactly an end user can use keyboard navigation.

Reviewer: The distinction is between the reader and the user - when we say 'user', it is about the user of the software or application, not the person reading the doc or instructions

Author: Okay, I will go with 'user' then.
